backend/project_review_coordination/models.py

```diff
@@ -33,7 +36,22 @@ def save(self, *args, **kwargs):
         super(ProjectReviewBundleClaim, self).save(*args, **kwargs)
 
     @staticmethod
-    def get_projects_user_can_review(user):
+    @lru_cache
+    def get_user_teams(user):
+        return user.teams()
```

Comment:
These functions seem like they are in a weird place. Should they not be on the User class since we are fetching information about the user? 

Why would anyone think to get the user permissions by checking the ProjectReviewBundleClaim static method? 

If we organise the code so that it is on the user class, would it not be much more useful? The only way the code could be reused if it is here would be:
- through spaghetti imports OR
- by first refactoring the code so that the functions are in a reusable place

Would it not be nice if this was available everwhere?

```
user.teams_cached()
user.get_teams_for_user_cached(perms=Team.PERMISSION_VIEW)
```

or even

```
user.teams(cached=True) 
user.get_teams_for_user(perms=Team.PERMISSION_VIEW,cached=True)
```